andela/firestar-backend
# #168490118 restructure Project directory to follow Regle standard

#### What does this PR do?
It restructure the project directory to match regle Folder structure so as to track developers activity
#### Description of Task to be completed?
None
#### How should this be manually tested?
This is a sample of the project folder @fodongkara suggested that the folder structure should look like, run `npm run test` to see that all test are passing
https://github.com/andela/shadowcat-backend
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#168490118](https://www.pivotaltracker.com/story/show/168490118)

#### Screenshots (if appropriate)
Test coverage:

![image](https://user-images.githubusercontent.com/38135488/64918329-9b2c1580-d794-11e9-9bda-449a8712ce04.png)


#### Questions:
